src/_scss/pages/bulkDownload/dictionary/_dictionaryTable.scss

```diff
@@ -17,23 +17,55 @@ $dictionary-orange-lighter: #FFFBF8;
 $dictionary-yellow-light: #FFFBD6;
 $dictionary-yellow: #FFF694;
 
-.dictionary-table {
-    .dictionary-table__message {
-        font-weight: $font-light;
-        text-align: center;
-        font-size: $h3-font-size;
-        padding-top: rem(60);
-        padding-bottom: rem(60);
-        td {
-            border: 0;
-        }
+::-webkit-scrollbar {
+    -webkit-appearance: none;
+    width: 7px;
+    height: 7px;
+}
+
+::-webkit-scrollbar-thumb {
+    border-radius: 4px;
+    background-color: rgba(0,0,0,.5);
+    -webkit-box-shadow: 0 0 1px rgba(255,255,255,.5);
+}
+
+.dictionary-table__above-scroller-hidden {
+    display: none;
+}
+
+.dictionary-table__above-scroller {
+    overflow-x: scroll;
+    height: rem(20);
+} 
+
+.dictionary-table__scroller {
+    height: rem(7);
+    width: rem(3024); /* This needs to be changed to the outer width of the inner table if width are not fixed */
+}
+
+.dictionary-table__message {
+    font-weight: $font-light;
+    text-align: center;
+    font-size: $h3-font-size;
+    padding-top: rem(60);
+    padding-bottom: rem(60);
+    td {
+        border: 0;
     }
+}
 
-    .dictionary-table__content {
-        border: none;
-        margin-top: 0;
+.dictionary-table__container {
+    height: rem(1050);
```

Comment:
Since the amount of units in the height is quite high, could this just be removed ( aka set it to "auto")?